src/features/swap/helpers/utils.ts

```diff
@@ -173,3 +173,18 @@ export const isOrderPartiallyFilled = (
 
   return BigInt(executedSellAmount) !== 0n && executedSellAmount < sellAmount
 }
+
+export const isSettingTwapFallbackHandler = (decodedData: DecodedDataResponse | undefined) => {
+  const isCow =
```

Comment:
nit: This variable assignment is not needed since the function name is already descriptive enough.